### PR TITLE
Remove assessment only funding

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Remove assessment only funding

### CODE REVIEW

File: get-into-teaching-app-pavel.code-workspace
Comment (EllieNodder) [inline get-into-teaching-app-pavel.code-workspace:1]: @plisovin Laura's asked me to review this PR - the content changes look good to me, but unsure what this change is for? Does this look okay to you?

Comment (plisovin) [reply get-into-teaching-app-pavel.code-workspace:1]: @EllieNodder and @laurahermionetennant92 this file is not needed here, you can add this to your git ignore or just delete from the commit

Comment (laurahermionetennant92) [reply get-into-teaching-app-pavel.code-workspace:1]: Hi @plisovin I'm sorry but I'm not sure what you mean – have you got 10 mins at some point today to talk me through it over Teams?

Comment (plisovin) [reply get-into-teaching-app-pavel.code-workspace:1]: @laurahermionetennant92 I mean you don't need to commit this file ```get-into-teaching-app-pavel.code-workspace```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**